server/app/lib/actions/fusor/deploy.rb

```diff
@@ -18,6 +18,11 @@ def humanized_name
       end
 
       def plan(deployment, skip_content = false)
+
+        # Create deployment specific log file.
+        Dir.mkdir("#{Rails.root}/log/#{deployment.name}")
```

Comment:
It looks like I can create deployments with names that have characters like `/` and `..`:

![screen shot 2015-11-19 at 2 43 43 pm](https://cloud.githubusercontent.com/assets/86290/11282171/f7747eba-8ecb-11e5-85d2-d1e6a407b55e.png)